samples/k8s-existing-cluster-only-sample/README.md

```diff
@@ -0,0 +1,33 @@
+---
+page_type: sample
+languages:
+- yaml
+- json
+products:
+- azure-kubernetes-service
+- azure-marketplace
+---
+
+# Existing cluster only sample
```

Comment:
To improve discoverability, can you please include this sample in the 2nd row in the table in repo's readme.md https://github.com/Azure-Samples/kubernetes-offer-samples#samples-in-this-repo ?